app/jobs/shipit/merge_merge_requests_job.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+module Shipit
+  class MergeMergeRequestsJob < BackgroundJob
```

Comment:
I just noticed this. `MergeMerge` is a bit weird. I'm not sure what would be better though. Maybe `ProcessMergeRequestsJob`?

Reply:
On a scale of 1-5, I'm a 5 (Strongly Agree). 😄

Reply:
Resolved in: https://github.com/Shopify/shipit-engine/pull/1095/commits/ce8120a5f1e10b478c6b0facf5893a2027974d60